packages/hardhat-core/test/internal/hardhat-network/helpers/retrieveForkBlockNumber.ts

```diff
@@ -1,10 +1,9 @@
-/* tslint:disable:no-string-literal */
 import { ForkBlockchain } from "../../../../src/internal/hardhat-network/provider/fork/ForkBlockchain";
 import { HardhatNetworkProvider } from "../../../../src/internal/hardhat-network/provider/provider";
 
 export async function retrieveForkBlockNumber(
   provider: HardhatNetworkProvider
-) {
+): Promise<number> {
```

Comment:
I had to do this because otherwise `@typescript-eslint/restrict-plus-operands` was giving me false positives in a lot of tests. I have no idea why, maybe it was a bug. Anyway, this fixed it :man_shrugging: